README.md

```diff
@@ -7,6 +7,12 @@
 
 A Julia interface for running a test FTP server with [pyftpdlib](https://pyftpdlib.readthedocs.io/en/latest/index.html).
 
+## Prerequisites
+
+FTPServer makes use of PyCall for running the FTP server.
+It's recommended that you use [`PYTHON=""`](https://github.com/JuliaPy/PyCall.jl#specifying-the-python-version) so that Julia uses it's own Python distribution.
+However, if you wish to specify a system Python you'll to at least use Python 3 and install the Python packages `pyopenssl` and `pyftpdlib`.
```

Comment:
3.6, 3.7, doesn't matter?

Reply:
I don't believe it matters but I can check some older PRs

Reply:
Looks like we didn't care then either: https://github.com/invenia/FTPServer.jl/pull/16